src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
```

Comment:
- [ ] The `App` component should be class-based, as per the project's requirement. Kindly make the required changes here. :pray:

Reply:
Well addressed this!